README.md

```diff
@@ -77,18 +77,19 @@ non-default fonts.
 
 ## Proposal
 
-Each browser will be able to map from a locale (as in [Goals](#goals), just the
-negotiated language+country pair, not the whole `Accept-Language` list) to:
+Each browser has a map from a [locale](#locale) to a set of fonts that it
+ensures are available locally. There are several options for which fonts are in
+this set and how they're used.
 
 ### Allowed system fonts
 
-This resembles Safari's list of fonts usable from CSS, which it receives from an
-OS API. Ideally each operating system should provide a system API to determine
-whether a given font is pre-installed or user-installed. Browsers will only
-allow use of pre-installed fonts in places like the `@font-face` `src:`
+Browsers only allow pre-installed fonts in places like the `@font-face` `src:`
 [`local()`](https://www.w3.org/TR/css-fonts-3/#font-face-name-value) function
 and the [`font-family`
-property](https://www.w3.org/TR/css-fonts-3/#font-family-prop).
+property](https://www.w3.org/TR/css-fonts-3/#font-family-prop). Safari currently
+does this by passing a bit to the OS font lookup routines that prevents them
+from finding user- or application-installed fonts. Apple intends to expose that
```

Comment:
Please change "intends to expose" to "is considering exposing."

Reply:
Done, thanks.